src/pint/models/binary_ell1.py

```diff
@@ -79,21 +79,25 @@ class BinaryELL1(PulsarBinary):
     ----------
     - Lange et al. (2001), MNRAS, 326 (1), 274–282 [1]_
     - Zhu et al. (2019), MNRAS, 482 (3), 3249-3260 [2]_
+    - Fiore et al. (2023), arXiv:2305.13624 [astro-ph.HE] [3]_
 
     .. [1] https://ui.adsabs.harvard.edu/abs/2019MNRAS.482.3249Z/abstract
     .. [2] https://ui.adsabs.harvard.edu/abs/2001MNRAS.326..274L/abstract
-
+    .. [3] https://arxiv.org/abs/2305.13624
 
     Notes
     -----
     This includes o(e^2) expression for Roemer delay from Norbert Wex and Weiwei Zhu
-    This is equaiton (1) of Zhu et al (2019) but with a corrected typo:
+    This is equation (1) of Zhu et al (2019) but with a corrected typo:
         In the first line of that equation, ex->e1 and ey->e2
         In the other lines, ex->e2 and ey->e1
     See Email from Norbert and Weiwei to David on 2019-Aug-08
     The dre expression comes from Norbert and Weiwei; the derivatives
     were calculated by hand for PINT
```

Comment:
Can you write full names here? 

It will be nice to have that email stored somewhere for posterity. (This may not be possible given privacy concerns.)

Reply:
This text was introduced with the o(e^2) terms, not by me...I don't actually know which David is meant here. It could be Kaplan but I think it is Nice? 

It does say "Norbert Wex" and "Weiwei Zhu" on line 90, and then it changes to first names. I could change it to full names all the way through if you think that is best.

Reply:
I think the email was to David Nice.

Reply:
ok, changed that to David Nice and changed "Norbert"/"Weiwei" to NW/WZ after the first mention of their full names. I think this clarifies things.